### PR TITLE
Add self signing step to MacOS instructions

### PR DESCRIPTION
**Pull request by stevietv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -625,7 +625,13 @@ <h4 class="alert-heading">Updating Sonarr v2 to v3</h4>
                                     In macOS 10.12+ Gatekeeper App Translocation will prevent Sonarr from updating if it's being run directly from the download location. Therefore it MUST be moved to the Application folder.
                                 </div>
                             </dd>
-                            <dt><span class="step">3</span>Start Sonarr</dt>
+                            <dt><span class="step">3</span>Self-sign Sonarr</dt>
```
Comment (markus101) [inline index.html:628]: This needs to be done in the relevant `.marko` file and then compiled with `yarn build` which will create `index.html`

**Review by markus101 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (stevietv) [reply index.html:628]: thanks for the tip. `macos.marko` is updated and `yarn build` executed

File: src/sections/downloads-v3/macos.marko [hunk 2]
```diff
@@ -23,15 +23,20 @@
             In macOS 10.12+ Gatekeeper App Translocation will prevent Sonarr from updating if it's being run directly from the download location. Therefore it MUST be moved to the Application folder.
         </alert>
     </install-step>
-    <install-step num="3" title="Start Sonarr">
+    <install-step num="3" title="Self-sign Sonarr">
+        <p>
+            Self-sign Sonarr <code>codesign --force --deep -s - Sonarr.app</code>
```
Comment (markus101) [inline src/sections/downloads-v3/macos.marko]:
```suggestion
            Open Terminal and run <code>codesign --force --deep -s - Sonarr.app</code>
```

**Review by markus101 — CHANGES_REQUESTED** [review]
Thanks for the changes, just one comment that I think will make it more explicit (and rather than repeating the step's text).

**Comment by markus101** [comment]
Thanks Stevie!

**Pull request merged**